tests/upload_test.go

```diff
@@ -415,38 +415,29 @@ var _ = Describe("[rfe_id:138][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			if utils.DefaultStorageClassCsiDriver == nil {
 				Skip("No CSI driver found")
 			}
-			verifyCleanup(pvc)
 		})
 
-		AfterEach(func() {
-			By("Deleting verifier pod")
-			err = utils.DeleteVerifierPod(f.K8sClient, f.Namespace.Name)
-			Expect(err).ToNot(HaveOccurred())
-
-			err := f.DynamicClient.Resource(uploadSourceGVR).Namespace(f.Namespace.Name).Delete(context.TODO(), "upload-populator-test", metav1.DeleteOptions{})
-			if err != nil && !k8serrors.IsNotFound(err) {
-				Expect(err).ToNot(HaveOccurred())
-			}
-
-			By("Delete upload population PVC")
-			err = f.DeletePVC(pvc)
-			Expect(err).ToNot(HaveOccurred())
-
-		})
 		Context("standard", func() {
 			BeforeEach(func() {
 				err := createUploadPopulatorCR(cdiv1.DataVolumeKubeVirt)
 				Expect(err).ToNot(HaveOccurred())
 			})
 
-			DescribeTable("should", func(uploader uploadFunc, validToken, blockMode bool, expectedStatus int) {
+			FDescribeTable("should", func(uploader uploadFunc, validToken, blockMode bool, expectedStatus int) {
 				pvcDef := utils.UploadPopulationPVCDefinition()
 				if blockMode {
 					if !f.IsBlockVolumeStorageClassAvailable() {
 						Skip("Storage Class for block volume is not available")
 					}
 					pvcDef = utils.UploadPopulationBlockPVCDefinition(f.BlockSCName)
 				}
+				// Let's see if the PVC exist before creating it
+				pvc, err = utils.FindPVC(f.K8sClient, f.Namespace.Name, pvcDef.Name)
+				if pvc != nil {
+					By("PVC already exists. This isn't good")
```

Comment:
if you look at the Get() impl. https://github.com/kubevirt/containerized-data-importer/blob/00844de42be3b5f65a5a6795b5cda9e9f0954cc9/vendor/k8s.io/client-go/kubernetes/typed/core/v1/persistentvolumeclaim.go#L74 the pvc result is never actually nil